inference-engine/src/mkldnn_plugin/nodes/batch_to_space.cpp

```diff
@@ -1,221 +1,306 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2020-2021 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
 
 #include "base.hpp"
+#include "ie_parallel.hpp"
 
 #include <cmath>
 #include <string>
 #include <vector>
-#include <cassert>
 #include <set>
-#include "ie_parallel.hpp"
+#include <cassert>
 
 namespace InferenceEngine {
 namespace Extensions {
 namespace Cpu {
 
 class BatchToSpaceImpl: public ExtLayerBase {
 public:
-    explicit BatchToSpaceImpl(const CNNLayer* layer) {
+    explicit BatchToSpaceImpl(const CNNLayer *layer) {
         try {
             const auto batchToSpaceLayer = dynamic_cast<const BatchToSpaceLayer*>(layer);
             if (!batchToSpaceLayer)
-                THROW_IE_EXCEPTION << "'" << layer->name << "' layer is not instance of BatchToSpaceLayer class";
+                THROW_IE_EXCEPTION << "BatchToSpace layer with name '" << layer->name << "' isn't instance of BatchToSpaceLayer class";
+
+            if (batchToSpaceLayer->insData.size() != 4)
+                THROW_IE_EXCEPTION << "BatchToSpace layer with name '" << batchToSpaceLayer->name << "' has incorrect number of input edges";
+
+            if (batchToSpaceLayer->outData.size() != 1)
+                THROW_IE_EXCEPTION << "BatchToSpace layer with name '" << batchToSpaceLayer->name << "' has incorrect number of output edges";
 
-            if (batchToSpaceLayer->insData.size() != 4 || batchToSpaceLayer->outData.size() != 1)
-                THROW_IE_EXCEPTION << "'" << batchToSpaceLayer->name << "' layer has incorrect number of input or output edges!";
+            auto data = batchToSpaceLayer->insData[0].lock();
+            if (!data)
+                THROW_IE_EXCEPTION << "BatchToSpace layer with name '" << batchToSpaceLayer->name << "' has nullable input data";
 
-            auto inData = batchToSpaceLayer->insData[0].lock();
-            if (inData == nullptr)
-                THROW_IE_EXCEPTION << "'" << batchToSpaceLayer->name << "' layer has nullable input data";
+            inDims = batchToSpaceLayer->insData[0].lock()->getTensorDesc().getDims();
```

Comment:
Minor comment:
```suggestion
            inDims =data->getTensorDesc().getDims();
```
Similarly for line 47:
`const auto precision = data->getTensorDesc().getPrecision();`